README.md

```diff
@@ -201,6 +201,32 @@ This will match the export name `"react-server"` and `"edge-light"` then use the
 
 `bunchee` has support for checking the package bundles are matched with package exports configuration.
 
+### Wildcard (Experimental)
+
+You can use wildcard `"./*"` to match all [exportable entries](https://github.com/huozhi/bunchee?tab=readme-ov-file#exportable-entries).
+
+```json
+{
+  "exports": {
+    ".": {
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.cjs"
+    },
+    "./*": {
+      "import": "./dist/*.mjs",
+      "require": "./dist/*.cjs"
+    }
+  }
+}
+```
+
+#### Exportable Entries
```

Comment:
extensions is a general support, we don't have to put it under wildcard, let's remove it for now and introduce it later separately

Reply:
So shall I remove `#### Exportable Entries` at all?